libs/blocks/form/form.js

```diff
@@ -232,7 +233,7 @@ function lowercaseKeys(obj) {
   }, {});
 }
 
-async function createForm(formURL, thankYou, formData) {
+async function createForm({ formURL, thankYou, formData, endpoint }) {
```

Comment:
Could we add a default to avoid `cannot access property of undefined`? As in
```suggestion
async function createForm({ formURL, thankYou, formData, endpoint } = {}) {
```